pybossa/forms/forms.py

```diff
@@ -269,6 +269,7 @@ def in_range(self):
 
 class TaskNotificationForm(Form):
     remaining = IntegerField(lazy_gettext('Notify when the number of remaining tasks is less than or equal to'))
+    webhook = TextField(description=lazy_gettext('Provide your webhook URL'))
```

Comment:
this could be more descriptive as "Webhook URL to send remaining. task notification."